assignments/objects.js

```diff
@@ -10,35 +10,73 @@
 // 5,adaine5@samsung.com,Antonietta,F
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
-let example = {
+let employee = {
   "id": 0,
   "name": "Example",
   "email": "examples@you.edu",
   "gender": "F"
 }
 
-// Write your intern objects here:
 
+// Write your intern objects here:
+let internOne = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+let internTwo = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+  }
+
+let internThree = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+let internFour = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+let internFive = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(internOne["name"]);
 // Kennan's ID
-
+console.log(internTwo["id"]);
 // Keven's email
-
+console.log(internThree["email"]);
 // Gannie's name
-
+console.log(internFour["name"]);
 // Antonietta's Gender
+console.log(internFive["gender"]);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+internTwo.speak = function() {
+  return `Hello, my name is ${internTwo.name} !`;
+}
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
+internFive.multiplyNums = function(a,b) {
```

Comment:
Nice job, you're code is well written and readable.  I should have mentioned yesterday as well, its generally best practices to put a space after a comma when dealing with multiple inputs in functions.  But it does not change functionality.  so it is a matter of personal preference.